### PR TITLE
update CHANGELOG

### PR DESCRIPTION
**Pull request by fspmarshall**
Update changelog in preparation for 4.4.3.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+### 4.4.3
+
+This release of teleport adds enhancements to the access request (workflow) API.
```
Comment (benarent) [inline CHANGELOG.md:5]:
```suggestion
This release of teleport adds enhancements to the Access Workflows API.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+* Support for creating limited roles that trigger access requests
+on login, allowing users to be configured s.t. no nodes can be accessed
+without externally granted roles.
+
+* Web UI support for automatically generating access requests and
```
Comment (benarent) [inline CHANGELOG.md:11]:
```suggestion
* Teleport UI support for automatically generating access requests and
```

**Review by benarent — APPROVED** [review]
Looks good, with the addition of my suggestions.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+### 4.4.3
+
+This release of teleport adds enhancements to the Access Workflows API.
```
Comment (webvictim) [inline CHANGELOG.md:5]:
```suggestion
This release of Teleport adds enhancements to the Access Workflows API.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+
+* Support for creating limited roles that trigger access requests
+on login, allowing users to be configured s.t. no nodes can be accessed
```
Comment (webvictim) [inline CHANGELOG.md:8]: What's `s.t.` here? "So that"? "Such that"? We should be explicit and use plain English in release notes.

**Review by webvictim — APPROVED** [review]

**Comment by webvictim** [comment]
LGTM pending comments

**Pull request merged**